### DISCUSSION TITLE
Cloud restore size problem

### DISCUSSION THREAD

**Discussion by aberiau** [ravendb/ravendb#12979] [Q&A]
Hi,

I've been working on a free license of raven 5.2 on my desktop. Now that i want to experiment with a cloud license, i want to transfer my database. Thing is, i dont have access to the snapshot functionality, so i need to make a backup. Once i'm done restoring my backup, raven starts processing the documents and indexes. It's been going on for almost a day now! My database is only 34gb it's never been that long before.

Is there a better way to upload my database? 
Should i just let it finish?
Will the temp files remove themselves or do i need to do something else and why do they take so much spaces?
Should i just pay for more ram?

![image](https://user-images.githubusercontent.com/93675316/140168006-98a4f91c-0da6-4d30-a90e-942237b161c1.png)

Thank you.

**Comment by garayx** [comment]
Hi 

Please use the [cloud portal](https://cloud.ravendb.net/portal/) to submit cloud related tickets,

Regarding the Temp folder it should get cleaned, yes.
